src/crimson/os/seastore/transaction.h

```diff
@@ -284,12 +298,18 @@ class Transaction {
     assert(extent.get_paddr() == placeholder.get_paddr());
     assert(extent.get_paddr().is_absolute());
     {
-      auto where = read_set.find(placeholder.get_paddr());
+      auto where = read_set.find(placeholder.get_paddr(), extent_cmp_t{});
       assert(where != read_set.end());
       assert(where->ref.get() == &placeholder);
       where = read_set.erase(where);
-      auto it = read_set.emplace_hint(where, this, &extent);
+      placeholder.read_transactions.erase(
+	read_trans_set_t<Transaction>::s_iterator_to(*where));
+      read_items.emplace_back(this, &extent);
```

Comment:
I assume the only changed place is here compared with the last version, right?

Reply:
Yes:-)